IaC/report-uptime.yml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: uptime-report
+    project-type: pipeline
+    description: |
+        report uptime of jenkins master
+        Do not edit this job through the web!
+    disabled: false
+    concurrent: false
+    properties:
+    - build-discarder:
+        days-to-keep: -1
+        num-to-keep: 10
+        artifact-days-to-keep: 0
+        artifact-num-to-keep: 0
+    triggers:
+    - timed: '* * * * *'
+    dsl: |
+        node('master') {
+          sh """
+            UNIT=\$(uptime -p | awk '{print \$3}')
```

Comment:
uptime -p
 14:06:50 up 29 days,  2:08,  load average: 0.71, 0.66, 0.67

so $3 = 29

Reply:
```
$ ssh ec2-user@rel.cd.percona.com
Last login: Wed Nov 11 12:09:59 2020 from 37.46.255.109

       __|  __|_  )
       _|  (     /   Amazon Linux 2 AMI
      ___|\___|___|

https://aws.amazon.com/amazon-linux-2/
No packages needed for security; 67 packages available
Run "sudo yum update" to apply all updates.
-bash: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory
[ec2-user@ip-10-199-1-212 ~]$ sudo -s
[root@ip-10-199-1-212 ec2-user]# uptime -p
up 20 hours, 57 minutes'
```

Reply:
uptime -p
up 2 years, 6 weeks, 5 days, 3 hours, 15 minutes

Your check should be universal and should not break on different time

echo $(awk '{print $1}' /proc/uptime) / 60 | bc - it will show uptime in minutes